src/foam/core/lib.js

```diff
@@ -70,6 +70,19 @@ foam.assert = function assert(cond) {
 };
 
 
+foam.getMethodName = (function named() {}).name === 'named' ?
```

Comment:
I think this should be part of the foam.Function library.

What if we move this to the stop of stdlib.js and define

foam.LIB({
  name: 'foam.Function',
  methods: [ { name: 'getName', code: ... } ]
});